docs/src/examples/hamiltonian_nn.md

```diff
@@ -57,7 +57,7 @@ res = Optimization.solve(opt_prob, opt, dataloader; callback)
 
 ps_trained = res.u
 
-model = NeuralHamiltonianDE(
+model = NeuralODE(
```

Comment:
Why was this changed away from Hamiltonian?

Reply:
https://github.com/SciML/DiffEqFlux.jl/pull/945/files/42fe746f412813b1f0bbdcbac1bc99e6c2bbe5fb#diff-8af080be8793cf24a571c91750a7b2432e3820c5c45f68f3f1d13937d044bfa9L36 constructs the hamiltonian. In the last release this function was mostly a no-op but I forgot to change it